acceptance/lib/puppet/acceptance/install_utils.rb

```diff
@@ -124,127 +124,39 @@ def install_repos_on(host, project, sha, repo_configs_dir)
-            on host, "rpm -Uvh --force #{repo_loc}/*.rpm"
-
-          when /^(debian|ubuntu)-([^-]+)-(.+)$/
-            variant = $1
-            version = $2
-            arch = $3
-
-            # If this isn't outdated yet it will be by end of week (4/3/2015)
-            deb = fetch(
-              "http://apt.puppetlabs.com/",
-              "puppetlabs-release-%s.deb" % version,
-              platform_configs_dir
-            )
-
-            list = fetch(
-              "http://%s/%s/%s/repo_configs/deb/" % [tld, project, sha],
-              "pl-%s%s-%s.list" % [project, sha ? '-' + sha : '', version],
-              platform_configs_dir
-            )
-
-            repo_dir = fetch_remote_dir("http://%s/%s/%s/repos/apt/%s" % [tld, project, sha, version], platform_configs_dir)
-            repo_loc = "/root/#{project}"
-
-            on host, "rm -rf #{repo_loc}"
-            on host, "mkdir -p #{repo_loc}"
-
-            scp_to host, deb, repo_loc
-            scp_to host, list, repo_loc
-            scp_to host, repo_dir, repo_loc
-
-            pc1_check = on(host,
-                           "[[ -d /root/#{project}/#{version}/pool/PC1 ]]",
-                           :acceptable_exit_codes => [0,1])
-
-            repo_name =  pc1_check.exit_code == 0 ? 'PC1' : 'main'
-            on host, "cp #{repo_loc}/*.list /etc/apt/sources.list.d"
-            on host, "find /etc/apt/sources.list.d/ -name \"*.list\" -exec sed -i \"s/deb\\s\\+http:\\/\\/#{tld}.*$/deb file:\\/\\/\\/root\\/#{project}\\/#{version} #{version} #{repo_name}/\" {} \\;"
-            on host, "dpkg -i --force-all #{repo_loc}/*.deb"
-            on host, "apt-get update"
-          else
-            host.logger.notify("No repository installation step for #{platform} yet...")
+        when /^(fedora|el|centos)-(\d+)-(.+)$/
+          variant = (($1 == 'centos') ? 'el' : $1)
+          fedora_prefix = ((variant == 'fedora') ? 'f' : '')
+          version = $2
+          arch = $3
+
+          repo_filename = "pl-%s%s-%s-%s%s-%s.repo" % [
+            project,
+            sha ? '-' + sha : '',
+            variant,
+            fedora_prefix,
+            version,
+            arch
+          ]
+          repo_url = "http://%s/%s/%s/repo_configs/rpm/%s" % [tld, project, sha, repo_filename]
+
+          on host, "curl -o /etc/yum.repos.d/#{repo_filename} #{repo_url}"
+        when /^(debian|ubuntu)-([^-]+)-(.+)$/
+          variant = $1
+          version = $2
+          arch = $3
+
+          list_filename = "pl-%s%s-%s.list" % [
+            project,
+            sha ? '-' + sha : '',
+            version
+          ]
+          list_url = "http://%s/%s/%s/repo_configs/deb/%s" % [tld, project, sha, list_filename]
+
+          on host, "curl -o /etc/apt/sources.list.d/#{list_filename} #{list_url}"
+          on host, "apt-get update"
```

Comment:
@justinstoller mentioned that we're not calling `yum makecache`, and likely don't need to since we're installing a new repo config, but something to consider.